src/features/hotspots/settings/WifiSetup.tsx

```diff
@@ -0,0 +1,93 @@
+import React, { useEffect, useState } from 'react'
+import { useTranslation } from 'react-i18next'
+import Box from '../../../components/Box'
+import Button from '../../../components/Button'
+import CircleLoader from '../../../components/CircleLoader'
+import Text from '../../../components/Text'
+import TextInput from '../../../components/TextInput'
+import { useConnectedHotspotContext } from '../../../providers/ConnectedHotspotProvider'
+import animateTransition from '../../../utils/animateTransition'
+import { hp } from '../../../utils/layout'
+import useAlert from '../../../utils/useAlert'
+import useKeyboard from '../../../utils/useKeyboardHeight'
```

Comment:
should this import be `useKeyboardHeight` instead of just `useKeyboard`?

Reply:
yeah. I think so.